plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Repositories.js

```diff
@@ -371,6 +375,46 @@ Ext.define('NX.coreui.controller.Repositories', {
           scope: button
         }
     );
+  },
+
+  /**
+   * @private
+   * Rebuild Repository index for the selected Repository.
+   */
+  rebuildIndex: function() {
+    var me = this,
+        model = me.getList().getSelectionModel().getLastSelected();
```

Comment:
@cstamas There was a bit more required here, whereas in the 'delete' example you were copying from the model lookup is down as part of Drilldown.js logic.
me.getList() : uses the defined 'refs' for list, which is just a shorthand lookup for 'nx-coreui-repository-list'
After that we just grab the model that was selected in that list and is depicted in the CRUD screen. If you look at the model for Repository.js, you'll see that the 'name' property is mapped as the ID so we can use the getId() method; alternatively you could use get('name') to be explicit about the property you're using, but that detail shouldn't be necessary for a use-case like this one IMO.
Feel free to ping me if any of this isn't clear.